src/routes/clinical/arranger.ts

```diff
@@ -0,0 +1,95 @@
+import { ADVERTISED_HOST } from 'config';
```

Comment:
naive question: is `config` an npm package or an internal module with an absolute path? if the latter, this looks weird in the same set of imports as the rest.